src/ui/pages/Networks/NetworkUpdateSuccess/NetworkUpdateSuccess.tsx

```diff
@@ -10,19 +10,25 @@ import { noValueDash } from 'src/ui/shared/typography';
 import { Button } from 'src/ui/ui-kit/Button';
 import { NavigationTitle } from 'src/ui/components/NavigationTitle';
 import type { NetworkConfig } from 'src/modules/networks/NetworkConfig';
+import { useNetworks } from 'src/modules/networks/useNetworks';
+import { createChain } from 'src/modules/networks/Chain';
+import { Networks } from 'src/modules/networks/Networks';
 import { ValueCell } from '../shared/ValueCell';
 
 export function NetworkUpdateSuccess({
   network,
-  oldNetwork,
+  prevNetwork,
   onClose,
 }: {
   network: NetworkConfig;
-  oldNetwork: NetworkConfig;
+  prevNetwork: NetworkConfig;
   onClose: () => void;
 }) {
+  const { networks } = useNetworks();
+
+  const prevRpcUrl = Networks.getNetworkRpcUrlInternal(prevNetwork);
+  const rpcUrl = networks?.getRpcUrlInternal(createChain(network.chain));
```

Comment:
Looks like having getRpcUrlInternal for network as a param is also nice, cause now we are converting network into chain just to convert it back inside the function

Reply:
Or may be we can just do `Networks.getNetworkRpcUrlInternal(network);` here?